style.sh

```diff
@@ -19,7 +19,7 @@ if [ -z $VIRTUAL_ENV ]; then
 fi
 
 printf "[ISORT: project-liberation]\n"
-${BASH_SOURCE%/*}/find-files-to-check.sh | xargs isort -sl -l 120
+${BASH_SOURCE%/*}/find-files-to-check.sh | xargs isort --sl -l 120
```

Comment:
=========================== CODING STYLE ===========================
[ISORT: project-liberation]
usage: isort [-h] [-a ADD_IMPORTS] [-ac] [-af] [-b KNOWN_STANDARD_LIBRARY]
             [-c] [-ca] [-cs] [-d] [-df] [-ds] [-dt] [-e]
             [-f KNOWN_FUTURE_LIBRARY] [-fas] [-fass] [-ff FROM_FIRST]
             [-fgw [FORCE_GRID_WRAP]] [-fss] [-i INDENT] [-j JOBS] [-k]
             [-l LINE_LENGTH] [-lai LINES_AFTER_IMPORTS]
             [-lbt LINES_BETWEEN_TYPES] [-le LINE_ENDING] [-ls]
             [-m {0,1,2,3,4,5,6,7}] [-nis] [-nlb NO_LINES_BEFORE]
             [-ns NOT_SKIP] [-o KNOWN_THIRD_PARTY] [-ot]
             [-p KNOWN_FIRST_PARTY] [-q] [-r] [-rm REMOVE_IMPORTS] [-rr] [-rc]
             [-s SKIP] [-sd DEFAULT_SECTION] [-sg SKIP_GLOB] [-sl]
             [-sp SETTINGS_PATH] [-t FORCE_TO_TOP] [-tc] [-up] [-v] [-vb]
             [--virtual-env VIRTUAL_ENV] [--conda-env CONDA_ENV] [-vn]
             [-w LINE_LENGTH] [-wl WRAP_LENGTH] [-ws] [-y] [--unsafe]
             [--case-sensitive] [--filter-files]
             [files [files ...]]
isort: error: unrecognized arguments: --sl

Please fix it